service/history/taskProcessor.go

```diff
@@ -125,15 +123,15 @@ func (t *taskProcessor) start() {
 		notificationChan := t.workerNotificationChans[i]
 		go t.taskWorker(notificationChan)
 	}
-	t.logger.Info("Timer queue task processor started.")
+	t.logger.Info("Queue task processor started.")
```

Comment:
Maybe remove "Queue" and just leave it as "task processor ..."? It sounds redundant.

Reply:
Upated